singer-io/tap-stripe
# Tdl 13149 sync payment intent

# Description of change
- Added a new stream payment_intents has been added to the tap.
- Schema for payment_intents has been added.
- The added schema for charge stream in the shared directory as it is being used by payment_intent schema.
- Updated integration test cases for payment_intents stream.

# Manual QA steps
 - Verify that schema and metadata are available in the catalog for payment_intents by running discover mode.
 - Verify that data for payment_intents was collected by running sync mode.
 - Verify that payment_intents respect a bookmark.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
